cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -421,29 +421,19 @@ func (c *Client) addMember(name string, peerAddrs string, isLearner bool) ([]Mem
 	var (
 		lastError   error
 		respMembers []*etcdserverpb.Member
-		learnerID   uint64
 		resp        *clientv3.MemberAddResponse
 	)
 	err = wait.PollUntilContextTimeout(context.Background(), constants.EtcdAPICallRetryInterval, constants.EtcdAPICallTimeout,
 		true, func(_ context.Context) (bool, error) {
 			ctx, cancel := context.WithTimeout(context.Background(), etcdTimeout)
 			defer cancel()
 			if isLearner {
```

Comment:
After we lock the FG to true as it is GA in v1.32 https://github.com/kubernetes/kubernetes/pull/126374/, the current only option is `AddMemberAsLearner` with isLearner=true.

Reply:
See https://github.com/kubernetes/kubeadm/issues/3111#issuecomment-2362866676.

Reply:
The associated clean up tasks should be done when the FG is removed. 
For example, remove the `isLearner bool` parameter for `addMember` func here.